pkg/registry/apis/provisioning/usage.go

```diff
@@ -4,48 +4,71 @@ import (
 	"context"
 	"fmt"
 
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/codes"
 	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apiserver/pkg/endpoints/request"
 
+	"github.com/grafana/grafana/pkg/apimachinery/identity"
 	"github.com/grafana/grafana/pkg/storage/unified/resourcepb"
 )
 
-func (b *APIBuilder) collectProvisioningStats(ctx context.Context) (map[string]any, error) {
+func (b *APIBuilder) collectProvisioningStats(ctx context.Context) (metrics map[string]any, err error) {
+	ctx, span := b.tracer.Start(ctx, "Provisioning.Usage.collectProvisioningStats")
+	defer func() {
+		span.SetStatus(codes.Error, fmt.Sprintf("failed to fetch provisioning usage stats: %v", err))
+		span.End()
+	}()
+
 	m := map[string]any{}
 	if b.unified == nil {
+		span.SetStatus(codes.Ok, "unified storage is not available")
 		return m, nil
 	}
 
+	// FIXME: hardcoded to "default" for now -- it works for single tenant deployments
+	// we could discover the set of valid namespaces, but that would count everything for
+	// each instance in cloud.
+	ns := "default"
+	ctx, _, err = identity.WithProvisioningIdentity(ctx, ns)
```

Comment:
🤔  I wonder if we should move this further up the stack -- this will not be the only thing that depends on an identity in context

Reply:
noted for when we have to make this multi-tenant :)